frontend/src/components/Nav/MiniMenu.vue

```diff
@@ -127,7 +127,7 @@ h2 {
   right: -20px;
   height: calc(100vh - 64px);
   overflow: hidden;
-  z-index: 99;
+  z-index: 98;
```

Comment:
can u make this a css var in the root var declarations and then apply the same to both so we don't accidentally do this again.

this would then become:

```css
z-index: var(--loading-overlay-z-index) - 1;
```

Reply:
I think I'll give each use of z-index a variable. Then they can all be changed in once place!